src/style.css

```
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+body {
+  margin-top: 20px;
+}
+
+::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.searchbar {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.searchbar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.searchbar:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: "Avenir-Heavy", sans-serif;
+}
+
+.big-searchbar {
+  margin-bottom: 20px;
+  height: 70px;
+  padding-left: 62px;
+  background-size: 19px 19px;
+  background-position: 26px center;
+}
+
+.big-searchbar:focus {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px 19px;
+  background-position: 26px center;
+}
+
+.small-searchbar {
+  height: 42px;
+  padding: 10px 0 11px 33px;
+  background-size: 11px 11px;
+  background-position-y: 14px;
+  background-position-x: 12px;
+}
+
+.small-searchbar:focus {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px 11px;
+  background-position-y: 14px;
+  background-position-x: 12px;
+}
```

similar to `.big-searchbar:focus {`
```suggestion
.small-searchbar:focus {
  background-size: 11px 11px;
  background-position: 15px center;
}
```